server/src/views/includes/foot.jade

```diff
@@ -39,6 +44,10 @@ script(type='text/javascript', src='/js/controllers/game.js')
 script(type='text/javascript', src='/js/controllers/auth.js')
 script(type='text/javascript', src='/js/controllers/profile.js')
 script(type='text/javascript', src='/js/init.js')
+script(type='text/javascript', src='/js/firbase.config.js')
+script(type='text/javascript', src='/js/controllers/chat.js')
```

Comment:
To maintain readability I feel the chat controller declaration can be moved up the script declaration chain so it can be alongside other controllers.